howitworks.md

```diff
@@ -22,7 +25,8 @@ access. You can ask your hosting provider to be sure.
 
 # With Shell Access
 
-We recommend that most people with shell access use the
+We recommend that most people with shell access (and
+[root privileges](https://en.wikipedia.org/wiki/Superuser)) use the
```

Comment:
Certbot actually does support non-root operation; it's just not super well documented (and maybe not quite easy enough to use). I'd rather leave this out, but keep the section below explaining the caveats.